crates/worker/src/partition/mod.rs

```diff
@@ -280,9 +284,10 @@ where
                 match res.as_ref() {
                     // run_inner never returns normally
                     Ok(_) => warn!("Shutting partition processor down because it stopped unexpectedly."),
-                    Err(ProcessorError::TrimGapEncountered { gap_to_lsn }) =>
+                    Err(ProcessorError::TrimGapEncountered { gap_to_lsn, sequence_number }) =>
                         info!(
                             trim_gap_to_lsn = ?gap_to_lsn,
+                            ?sequence_number,
                             "Shutting partition processor down because it encountered a trim gap in the log."
                         ),
                     Err(err) => warn!("Shutting partition processor down because of error: {err}"),
```

Comment:
Would it be useful to log the raw offset somewhere around here, @AhmedSoliman?

Reply:
The sequencer number is sufficient really.